### PR TITLE
To skip the command output with Unrecognised field error

### PR DESCRIPTION
**Pull request by chenlizhong**

### CODE REVIEW

File: insights/parsers/lvm.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
 from .. import add_filter
 from .. import Parser, parser, get_active_lines, LegacyItemAccess
 from . import parse_fixed_table
+from insights.core.dr import SkipComponent
```
Comment (xiangce) [inline insights/parsers/lvm.py]: Seems SkipException is more suitable than SkipComponent. See http://insights-core.readthedocs.io/en/latest/exception_model.html#skipcomponent-and-skipexception

Comment (chenlizhong) [reply insights/parsers/lvm.py]: Below is the code in dr.py,  Only SkipComponent is pass exception. All other exceptions will be logged into log file. I do not want to log anything for this as this is normal for some lvm version does not support one or two fields.

~~~
       try:
            if component not in broker and component in DELEGATES and ENABLED[component]:
                log.info("Trying %s" % get_name(component))
                result = DELEGATES[component](broker)
                broker[component] = result
        except MissingRequirements as mr:
            if log.isEnabledFor(logging.DEBUG):
                name = get_name(component)
                reqs = stringify_requirements(mr.requirements)
                log.debug("%s missing requirements %s" % (name, reqs))
            broker.add_exception(component, mr)
        except SkipComponent:
            pass
        except Exception as ex:
            log.exception(ex)
            broker.add_exception(component, ex, traceback.format_exc())
~~~

Comment (chenlizhong) [reply insights/parsers/lvm.py]: Yes

File: insights/parsers/lvm.py [hunk 2]
```diff
@@ -92,6 +93,8 @@ class Lvm(Parser):
     """Base class for parsing LVM data in key=value format."""
 
     def parse_content(self, content):
+        if "Unrecognised field:" in content[-1]:
```
Comment (bfahr) [inline insights/parsers/lvm.py]: @chenlizhong I'm wondering if there should be a more generalized solution to this problem?  This message string is very specific and to some extent localized.

Comment (chenlizhong) [reply insights/parsers/lvm.py]: @bfahr 
Look at the source code of lvm.  "Unrecognised field:" is hardcode.  I also tested vgs, pvs, lvs they all give "Unrecognised field:" when the -o parameter following unsupport field.

~~~
if (!_field_match(rh, ws, (size_t) (we - ws), report_type_only)) {
    _display_fields(rh, 1, 0);
    log_warn(" ");
    log_error("Unrecognised field: %.*s", (int) (we - ws), ws);
~~~

https://github.com/lvmteam/lvm2/search?utf8=%E2%9C%93&q=Unrecognised+field&type=

Comment (bfahr) [reply insights/parsers/lvm.py]: Thank you @chenlizhong, that seems more reasonable then.

Comment (xiangce) [reply insights/parsers/lvm.py]: @chenlizhong - how about changing to ParseException?

**Pull request merged**